packages/react/src/recipes/FeaturePreviewLPs/FeaturePreviewLevelOne/components/FormExample.tsx

```diff
@@ -25,6 +26,20 @@ export function FormExample({type = 'default', colorMode = ColorModesEnum.LIGHT}
     <form>
       <>
         <Stack direction="vertical" padding="none" gap={24}>
+          <Box
+            borderWidth="thin"
+            borderStyle="solid"
+            borderColor="default"
+            backgroundColor="subtle"
```

Comment:
@nsolerieu Primer Brand has a slightly different gray tone for secondary backgrounds. We can do an alignment exercise separately, as it would impact many other components. Are you okay to go with this color for now?